public/chore.js

```diff
@@ -29,17 +29,17 @@ choreList.forEach(function (chore) {
     list.appendChild(li);
 });
 
-const app = document.querySelector('#app');
-app.appendChild(list);
-
 
-function addChore() {
+function addChore() { 
+    const newLi = document.createElement('li');
     const newChore = {
         name: document.getElementById('chore').value,
         assignee: document.getElementById('assigned').value
     }
-        choreList.push(newChore);
-        console.log(choreList);
+    newLi.textContent = `Chore: ${newChore.name} - Assigned to: ${newChore.assignee}`;
+    // choreList.push(newChore);
```

Comment:
Uncomment this out. Although we aren't using it yet, we will eventually want to use the choreList with all the new chores when we post it to the database.